tests/units/Glpi/Dashboard/Provider.php

```diff
@@ -250,10 +281,20 @@ public function testNbTicketsByAgreementStatusAndTechnicianGroup()
         $this->string($output['data']['series'][2]['name'])->isEqualTo('Late own');
         $this->string($output['data']['series'][3]['name'])->isEqualTo('On time');
 
+        $this->integer(count($output['data']['series'][0]['data']))->isEqualTo(2);
+        $this->integer(count($output['data']['series'][1]['data']))->isEqualTo(2);
+        $this->integer(count($output['data']['series'][2]['data']))->isEqualTo(2);
+        $this->integer(count($output['data']['series'][3]['data']))->isEqualTo(2);
+
         $this->integer($output['data']['series'][0]['data'][0])->isEqualTo(0);
         $this->integer($output['data']['series'][1]['data'][0])->isEqualTo(0);
         $this->integer($output['data']['series'][2]['data'][0])->isEqualTo(0);
-        $this->integer($output['data']['series'][3]['data'][0])->isEqualTo(1);
+        $this->integer($output['data']['series'][3]['data'][0])->isEqualTo(2);
+
+        $this->integer($output['data']['series'][0]['data'][1])->isEqualTo(0);
+        $this->integer($output['data']['series'][1]['data'][1])->isEqualTo(0);
+        $this->integer($output['data']['series'][2]['data'][1])->isEqualTo(0);
+        $this->integer($output['data']['series'][3]['data'][1])->isEqualTo(1);
 
```

Comment:
This second group data should be checked also for following `nbTicketsByAgreementStatusAndTechnicianGroup()` calls.

Also, maybe the test could be easier to read with something like:
```php
$expected = [
    'data' => [
        'labels' => [...],
        'series' => [
            [
                'name' => 'On time'
                'data'   => [
                    2,
                    1,
                ]
            ]
        ],
    ],
];
$this->array($output)->isEqualTo($expected);
```